.codeclimate.yml

```diff
@@ -34,6 +34,12 @@ engines:
   rubocop:
     enabled: true
     config: '.rubocop_cc.yml'
+prepare:
+  fetch:
+  - url: "https://raw.githubusercontent.com/ManageIQ/manageiq/master/.rubocop_base.yml"
+    path: ".rubocop_base.yml"
+  - url: "https://raw.githubusercontent.com/ManageIQ/manageiq/master/.rubocop_cc.yml"
+    path: ".rubocop_cc.yml"
```

Comment:
Ok, I'm following, so we fetch the _base and _cc from the main repo as part of the codeclimate run... we tell codeclimate to use .rubocop_cc.yml which pulls in the now local .rubocop_base.yml fetched from the main repo.

What do we do about provider changes to rubocop.yml?